discovery-provider/src/queries/get_balances.py

```diff
@@ -84,15 +83,15 @@ def get_balances(session, redis, user_ids, is_verified_ids_set=None):
     needs_refresh = [
         user_balance.user_id
         for user_balance in query
-        if does_user_balance_need_refresh(user_balance, False)
+        if does_user_balance_need_refresh(user_balance)
     ]
 
     verified_ids_set = is_verified_ids_set if is_verified_ids_set else set()
```

Comment:
I think we could lose this verified bit here too. Let's just give everybody the 12 hr refresh